src/stories/Library/review/review.scss

```
@@ -0,0 +1,36 @@
+.reviews {
+  margin: 0 $s-4xl 0 $s-4xl;
+}
+
+.review {
+  margin-bottom: $s-xl;
+  &:last-of-type {
+    margin-bottom: $s-2xl;
+  }
+
+  &__meta {
+    color: $c-text-secondary-gray;
+  }
+
+  &__heart {
+    // in Figma the heart is 12px but the img SVG in the browser has
+    // some white space in the top, so I went for making it visually
+    // the same height as the font to match the design brief
+    height: 14px;
```

```suggestion
    // Ensure SVG has same height as font despite top whitespace.
    height: 14px;
```

I appreciate the commentary but at the code level we may be able to short things down a bit.

---

👍 ❤️